docs/s3_test_results.md

```diff
@@ -383,17 +383,17 @@ Compatibility: 9/6/8 out of 11
 
 ## Versioning
 
-Compatibility: 16/19/24 out of 26
```

Comment:
As I see this PR fixes the test `test_versioning_bucket_multipart_upload_return_version_id` and deletion of buckets (#320) too, maybe we should update test result here?

Reply:
This test was fixed early https://github.com/nspcc-dev/neofs-s3-gw/pull/321

Reply:
Okay, but anyway the PR closes #320 too.